flexible_event_config.md

```diff
@@ -20,10 +51,82 @@ In general, the approach here is to more flexibly encode the output of the API f
 * Allow for choosing flexible reporting windows ([issue #46](https://github.com/WICG/attribution-reporting-api/issues/46) and [issue #736](https://github.com/WICG/attribution-reporting-api/issues/736))
 * Allow for setting the possible output space for a source based on a _prior probability distribution_ the ad tech has about the types of trigger that are possible, taking advantage of recent research into label differential privacy for [binary classification](https://arxiv.org/abs/2102.06062) and [regression problems](https://arxiv.org/abs/2212.06074).
 
+## Phase 1: Lite Flexible Event Level
+
+### API Changes
+
+We will add the following two optional parameters to the JSON in `Attribution-Reporting-Register-Source`: `max_event_level_reports` and `event_report_windows`:
+
+```jsonc
+{
+  ...
+  // Optional. This is a parameter that acts across all trigger types for the lifetime of this source. 
+  // It restricts the total number of event-level reports that this source can generate. 
+  // After this maximum is hit, the source is no longer capable of producing any new data. 
+  // The use of priority in the trigger attribution algorithm in the case of multiple attributable triggers remains unchanged. 
+  // Defaults to 3 for navigation sources and 1 for event sources
+  "max_event_level_reports": <int>,
+
+  // Optional. Represents a series of time windows, starting at <start time>.
+  // Reports for this source will be delivered an hour after the end of each window.
+  // Time is encoded as seconds after source registration.
+  // If event_report_windows is omitted, will use the default windows.
+  // End time is exclusive.
+  "event_report_windows": {
+    "start_time": <int>, // Optional, defaults to 0.
```

Comment:
I thought this was no longer being proposed for inclusion in phase 1?

Reply:
I have removed "start_time"